### DISCUSSION TITLE
Rename "master" to "main"

### DISCUSSION THREAD

**Discussion by dcuccia** [VirtualPhotonics/VTS#73] [Ideas]
Per [here](https://www.hanselman.com/blog/easily-rename-your-git-default-branch-from-master-to-main), probably a good thing to rename the branch `master` to `main` at some point. There's a nice helper Gist [here](https://gist.github.com/danieldogeanu/739f88ea5312aaa23180e162e3ae89ab).

**Comment by lmalenfant** [comment]
I actually did this in some of my projects and with all new projects, I definitely follow this guideline. However, I did get burned because if there are external factors involved, like pipelines, they might break by changing the name. 

Possibly not an issue for us but I need to do a little more research before we make the switch.
